packages/forms-web-app/src/views/register/agent/cya/cya-representee-detail.njk

```diff
@@ -19,7 +19,104 @@
 {% endif %}
 
 <h2 class="govuk-heading-m">Who you are representing?</h2>
-{{ govukSummaryList({
```

Comment:
Is there a way to build up the rows dynamically in Nunjucks?  That way we could call `govukSummaryList` just once with the right rows data.